addons/account/models/account_report.py

```diff
@@ -307,7 +307,7 @@ class AccountReportLine(models.Model):
     aggregation_formula = fields.Char(string="Aggregation Formula Shortcut", help="Internal field to shorten expression_ids creation for the aggregation engine", inverse='_inverse_aggregation_formula', store=False)
 
     _sql_constraints = [
-        ('code_uniq', 'unique (code)', "A report line with the same code already exists."),
+        ('code_uniq', 'unique (report_id, code)', "A report line with the same code already exists."),
```

Comment:
@oco-odoo It would be nice to be able to specify the id of the report for cross report formula to avoid the issue mentioned in that commit.

> Currently we require the account lines `code` field to be unique across
all reports in a database. Since Reportalypse, this is not required
anymore, since we search by default only in the current report unless
we specify a `cross_report` subformula. Then it will take the first one
it finds.

Reply:
Maybe some day. Though that feature is not used that often, so there are probably other things to do first.